config.toml

```
@@ -442,15 +441,16 @@ language_alternatives = ["en"]
 
 [languages.hi]
 title = "Kubernetes"
-description = "Production-Grade Container Orchestration"
-languageName = "हिन्दी (Hindi)"
+description = "प्रोडक्शन-ग्रेड कंटेनर ऑर्केस्ट्रेशन"
+languageName = "हिन्दी"
```

```suggestion
languageName = "हिन्दी (Hindi)"
```
- Main has been updated to follow the [language name standard](https://github.com/kubernetes/website/blob/main/content/en/docs/contribute/localization.md#modify-the-site-configuration). Also see #31657
- We should update this line in `dev-1.22-hi.1` so it is the same as `main`.

---

I have opened an issue and PR to fix this in `dev-1.22-hi.1`:
- https://github.com/kubernetes/website/issues/35783
- https://github.com/kubernetes/website/pull/35784

---

I have closed my PR in favor of @xzyangNB's PR, https://github.com/kubernetes/website/pull/35898, which resolves this issue.